ACE/ace/QtReactor/QtReactor.h

```diff
@@ -29,10 +29,12 @@
 
 // QT toolkit specific includes.
 #ifdef ACE_HAS_QT5
-#include /**/ <QtWidgets/QApplication>
+#include /**/ <QtCore/QCoreApplication>
+#define QAPPLICATION_TYPE QCoreApplication
```

Comment:
Does this need to be a preprocessor macro or could it be a typedef?  Either way, start the identifier with `ACE_` to avoid name collisions.

Reply:
Done